src/rgw/rgw_rest_s3.cc

```diff
@@ -3574,18 +3574,25 @@ RGWOp *RGWHandler_REST_Service_S3::op_post()
 RGWOp *RGWHandler_REST_Bucket_S3::get_obj_op(bool get_data)
 {
   // Non-website mode
-  int list_type = 1;
+  int list_type;
```

Comment:
i think it's always good to initialize a local variable with known value. even just for silencing the static analyzers.